content/applications/productivity/knowledge/properties.rst

```diff
@@ -2,101 +2,79 @@
+To add property fields, follow these steps:
+
+#. Click the :icon:`fa-ellipsis-v` (:guilabel:`ellipsis`) icon, then :menuselection:`Add Properties
+   --> Add a Property`.
+#. Type in the :guilabel:`label`.
+#. Select a :ref:`field type <knowledge/properties/field-types>`.
+#. Add a :guilabel:`default value`.
+#. (Optional) Tick the :guilabel:`Display in Cards` checkbox to show the properties in an
+   :ref:`article item's view <knowledge/articles_editing/commands>`.
+
+.. _knowledge/properties/field-types:
+
+Field types
+-----------
+
+The following field types are available:
+
+- :guilabel:`Text`
+- :guilabel:`Checkbox`
+- :guilabel:`Integer`
+- :guilabel:`Decimal`
+- :guilabel:`Date`
+- :guilabel:`Date & Time`
+
+The following field types need configuring:
+
+- :guilabel:`Selection`: To add a drop-down selection menu with choices, click :guilabel:`Add a
+  Value`, type the name of the choice, and press `Enter` to save it. Repeat this for each value.
+- :guilabel:`Tags`: To create tags, type a tag name in the :guilabel:`Tag` field and press `Enter`
+  or click :guilabel:`Create "new_tag"` to save it. Click on a tag to select a different color for
+  it.
+- :guilabel:`Separator`: Create a section between properties.
+
+The :ref:`Many2one <studio/fields/relational-fields-many2one>` and :ref:`Many2many
+<studio/fields/relational-fields-many2many>` field types are relational fields that allow the
+selection of one or multiple records from a model:
+
+#. Select one (**Many2one**) or multiple (**Many2many**) records from a different model as the field
+   value.
+#. Configure by clicking inside the :guilabel:`Model` field, then select the model type from which
+   to choose a record.
+#. After saving the property, click inside the property field to manually select values.
+
+.. note::
+   - Click outside the property field window to save a property.
+   - With the :guilabel:`Many2one` and :guilabel:`Many2many` field types, editing the
+     :ref:`Domain <search/custom-filters>` restricts the choice of values, while adding a
+     :guilabel:`Default value` suggests preselected values.
+
+.. tip::
+   Hover over the property name and click the :icon:`fa-pencil` (:guilabel:`pencil`) icon to edit
+   it or the :icon:`oi-draggable` (:guilabel:`drag`) icon to move it above or below another
+   property.
 
 Delete property fields
 ======================
 
-To remove a property, click the **pencil** icon next to the targeted property, then click
-:menuselection:`Delete --> Delete`.
+To remove a property, hover over its name, click the :icon:`fa-pencil` (:guilabel:`pencil`)
+icon, then click :menuselection:`Delete --> Delete`.
 
-.. warning::
-   Once a property field is deleted, you cannot retrieve it.
+.. note::
+   Deleting a property is permanent, and deleting all properties removes the property sidebar
+   panel.
 
 Hide the property panel
 =======================
```

Comment:
```suggestion
```
A tip at the bottom will suffice - it's not really worth having under a separate H2 heading, what do you think?